invenio_rdm_records/resources.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+# Copyright (C) 2020 Northwestern University.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Bibliographic Record Resource."""
+
+from invenio_records_resources.resources import RecordResource, \
+    RecordResourceConfig
+from invenio_records_resources.responses import RecordResponse
+from invenio_records_resources.serializers import RecordJSONSerializer
+
+from .marshmallow.json import RecordSchemaV1
+
+
+class BibliographicRecordResourceConfig(RecordResourceConfig):
+    """Bibliographic Record resource config."""
+
+    item_route = "/rdm-records/<pid_value>"
+    list_route = "/rdm-records"
+    response_handlers = {
+        "application/json": RecordResponse(
+            RecordJSONSerializer(schema=RecordSchemaV1)
+        )
+    }
+
+
+class BibliographicRecordResource(RecordResource):
```

Comment:
Two things:
1- registration/creation is going to be done in invenio-app-rdm because invenio-app-rdm's responsibility is to configure the rest of the modules (there we can order configuration for instance...). It also learns from the problem of invenio-rest-records and company who were registering themselves when installed, thus preventing the modules using them to fully control *how* they were going to use them; even if you only needed part of the functionality they provided, you were suddenly stuck with their endpoints. We don't want to reproduce that.

In turn, this means that invenio-app-rdm needs a prepared resource it can register. `BibliographicRecordResource` is that prepared resource.

2- Similar to the named services, we want a named Resource, so that in the shell it can be accessed directly. (same argument as for named Services)

Both of these points I understood from discussion after standup last week, so for sure we can double-check with @lnielsen 
   
This is something that @lnielsen clarified last week:

Reply:
Discussed IRL. All good.